internal/credential/store.go

```diff
@@ -16,114 +16,25 @@ limitations under the License.
 package credential
 
 import (
-	"context"
-	"fmt"
-	"os"
-
-	"github.com/docker/cli/cli/config"
-	"github.com/docker/cli/cli/config/configfile"
-	"github.com/docker/cli/cli/config/credentials"
-	"github.com/docker/cli/cli/config/types"
-	"oras.land/oras-go/v2/registry/remote/auth"
+	credentials "github.com/oras-project/oras-credentials-go"
```

Comment:
Is this `credentials` alias auto generated by the go tools?

Reply:
yes